src/radical/pilot/bootstrapper/default_bootstrapper.sh

```diff
@@ -1414,15 +1414,31 @@ profile_event 'cleanup done'
 echo "#"
 echo "# -------------------------------------------------------------------"
 
-echo
-echo "# -------------------------------------------------------------------"
-echo "#"
-echo "# Tarring profiles ..."
-PROFILES_TARBALL="profiles.tgz"
-tar -czf $PROFILES_TARBALL *.prof
-ls -l $PROFILES_TARBALL
-echo "#"
-echo "# -------------------------------------------------------------------"
+if ! test -z "`ls *.prof 2>/dev/null`"
```

Comment:
If all equal I couldn't care much either, I agree. But I was wondering whether this could break with gazillion of files.